Adafruit_Microbit.cpp

```diff
@@ -93,12 +93,6 @@ boolean Adafruit_Microbit_Matrix::begin(void) {
     digitalWrite(rowpins[r], LOW);
   }
 
-  for (uint8_t r = 0; r < MATRIX_ROWS; r++) {
-    for (uint8_t c = 0; c < MATRIX_COLS; c++) {
-      matrix_buffer[r][c] = 0;
```

Comment:
Already done in:
https://github.com/adafruit/Adafruit_Microbit/blob/e00bc76b38d86c09ae2e12d7236f8cea43ab7d1c/Adafruit_Microbit.cpp#L73-L75

Reply:
dont remove this code

Reply:
I've removed this commit from the PR, but isn't this buffer already set to zero by the time this code is executed?

Reply:
begin() should always 'reset' the board :)

Reply:
Ah, very good point! 👍